### PR TITLE
fix memory leak in rebind_symbols_image

### PR DESCRIPTION
**Pull request by rosen0510**
fix memory leak in rebind_symbols_image

### CODE REVIEW

File: fishhook.c [hunk 1]
```diff
@@ -187,6 +187,9 @@ int rebind_symbols_image(void *header,
     struct rebindings_entry *rebindings_head = NULL;
     int retval = prepend_rebindings(&rebindings_head, rebindings, rebindings_nel);
     rebind_symbols_for_image(rebindings_head, (const struct mach_header *) header, slide);
+    if(rebindings_head->rebindings){
```
Comment (dinhviethoa) [inline fishhook.c]:
```
if (rebindings_head->rebindings) {
```

**Comment by dinhviethoa** [comment]
Could you please format the code properly? Thanks!

**Comment by dinhviethoa** [comment]
Now, you just need to sign the CLA.

**Comment by rosen0510** [comment]
already signed now

**Pull request merged**